tools/dotcms-cli/cli/src/test/java/com/dotcms/cli/command/files/FilesPullCommandIntegrationTest.java

```diff
@@ -144,121 +146,7 @@ void Test_Command_Files_Pull_Option_Recursive2() throws IOException {
             commandLine.setOut(out);
             final String path = String.format("//%s", "default");
             final int status = commandLine.execute(FilesCommand.NAME,
-                    FilesPull.NAME, path, tempFolder.toString(), "--recursive");
-            Assertions.assertEquals(CommandLine.ExitCode.OK, status);
-        } finally {
-            deleteTempDirectory(tempFolder);
-        }
-    }
-
-    @Test
-    void Test_Command_Files_Pull_Option_Recursive3() throws IOException {
```

Comment:
why aren't these tests required anymore?

Reply:
On those tests I was testing the command didn't fail with the use of boolean after some option flags (`--some-option true` / `--some-option false`, with the improvements on the pull command flags and the validation of the proper use of the command using the `throwIfUnmatchedArguments` those boolean after the option are no longer valid.